src/new-client/src/index.tsx

```diff
@@ -1,16 +1,5 @@
 import { GA_TRACKING_ID } from "./constants"
-
-async function main() {
-  if (__TARGET__ === "web") {
-    await import("@/Web")
-  } else if (__TARGET__ === "openfin") {
-    await import("@/OpenFin")
-  } else if (__TARGET__ === "finsemble") {
```

Comment:
need to add a main for finsemble I think

Reply:
oops, added